### ISSUE TITLE
What modifications are made to WebNLG dataset for the experiments in the paper.

### ISSUE THREAD

**Issue by YuejiYang** [Yaoming95/Triple-to-Text#3]
Hello Yaoming,
Thanks for the paper. After reading the paper, I noticed you used WebNLG datatset. 
In the experiment section, as the paper claimed "We select some data in the three data sets and divide them into a training set and a test set".
The resulting WebNLG does not match any published WebNLG dataset for training and testing.
So, I wonder if there is any clarification on how you process the WebNLG dataset?

Thanks.

**Comment by YuejiYang** [comment]
There are seen and unseen categories for testing. It is quite difficult to obtain good performance on unseen test data that never appear in training data. So, I wonder by selection as claimed in the paper, how many unseen tests are included in the final test data?
